France-ioi/AlgoreaFrontend
# fix observation bar watched group link

## Done

Add `path` to `watchedGroup` to be able to redirect to it with its tree/ancestors

## Tests

1) Go to [a group page (branch app)](https://dev.algorea.org/branch/fix/observation-bar-group-link/en/#/groups/by-id/672913018859223173;path=52767158366271444/details)
2) Observe the group
3) You'll see at the top of the screen `"Observation mode: [group name]"`. Hover the group name link. You should see the path in the url, which is not the case in [production](https://dev.algorea.org/en/#/groups/by-id/672913018859223173;path=52767158366271444/details)
